### PR TITLE
Playwright_Pepper-1280-Add Blood and RNA Kit Upload for Family Member

### PR DESCRIPTION
**Pull request by Rinchan20**
Covers ticket: https://broadworkbench.atlassian.net/browse/PEPPER-1280 - but basically adds a blood and rna kit for a family member after the main study participant has a blood and rna kit uploaded

### CODE REVIEW

File: playwright-e2e/tests/rgp/dsm-family-enrollment.spec.ts [hunk 1]
```diff
@@ -551,7 +551,7 @@ test.describe.serial('DSM Family Enrollment Handling', () => {
 
     //Setup new family member
     const grandfather = new FamilyMemberTab(page, FamilyMember.MATERNAL_GRANDFATHER);
-    await grandfather.setRandomRelationshipID();
```
Comment (Rinchan20) [inline playwright-e2e/tests/rgp/dsm-family-enrollment.spec.ts:554]: I cannot remember why I initially made this randomized - but now with a test expecting a particular relationship id, changed it to using the set one

File: playwright-e2e/dsm/pages/participant-list-page.ts [hunk 2]
```diff
@@ -362,9 +363,20 @@ export default class ParticipantListPage extends DsmPageBase {
...
+            continue; //Looking for a family that has more than 1 person in the study
+          } else if (rgpMinimumFamilySize > 1 && amountOfFamilyMembers > 1) {
+            //Use the family member added to the account after the proband
+            shortID = JSON.stringify(value.esData.profile.hruid).replace(/['"]+/g, '');
+            logInfo(`Found RGP participant with Short ID: ${shortID} to have proband tabs - has ${amountOfFamilyMembers} study participants`);
+            return shortID;
+          } else if (rgpMinimumFamilySize === 1 && amountOfFamilyMembers === 1) {
+            shortID = JSON.stringify(value.esData.profile.hruid).replace(/['"]+/g, '');
+            logInfo(`Found RGP participant with Short ID: ${shortID} to have proband tabs - has 1 study participant`);
+            return shortID;
```
Comment (zyme) [inline playwright-e2e/dsm/pages/participant-list-page.ts:381]: I think you can move the `return shortID` down to the bottom of the conditional block and avoid repeating it here and on L373.

File: playwright-e2e/dsm/pages/rgp-participant-page.ts [hunk 3]
```diff
@@ -116,4 +118,50 @@ export default class RgpParticipantPage extends ParticipantPage {
     public getExpectedNumberToSequence(): Locator {
         return this.page.locator("//td[contains(text(), 'Expected # to Sequence')]/following-sibling::td/mat-select");
     }
+
+    public async getAmountOfFamilyMembers(): Promise<number> {
```
Comment (zyme) [inline playwright-e2e/dsm/pages/rgp-participant-page.ts]: Maybe `getNumberOfFamilyMembers` instead of amount?

**Review by zyme — APPROVED** [review]

**Comment by Rinchan20** [comment]
checked that it passes in test and dev - will merge once github checks are done

**Pull request merged**